lib/web_ui/skwasm/shaders.cpp

```diff
@@ -136,3 +137,21 @@ SKWASM_EXPORT SkShader* shader_createRuntimeEffectShader(
                    childPointers.data(), childCount, nullptr)
       .release();
 }
+
+SKWASM_EXPORT SkShader* shader_createFromImage(SkImage* image,
+                                               SkTileMode tileModeX,
+                                               SkTileMode tileModeY,
+                                               FilterQuality quality,
+                                               SkScalar* matrix33) {
+  if (matrix33) {
```

Comment:
Do we need to check? We seem to always pass a non-null matrix.

Reply:
I think I'm going to go the other direction and bubble the nullability up to the higher level APIs. There is at least one place I could avoid passing this data altogether.